components/script/devtools.rs

```diff
@@ -65,7 +67,7 @@ pub(crate) fn handle_evaluate_js(
             ScriptFetchOptions::default_classic_script(global),
             global.api_base_url(),
             can_gc,
-            None,
+            Some(c"debugger eval"),
```

Comment:
Can we use a named constant here?

Reply:
done in 5e961fc581ba2083e209934b1d39a1ab5a8eac08